### PR TITLE
Includes the installed module profiles

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

**Comment by parthaa** [comment]
check w.r.t https://github.com/candlepin/subscription-manager/pull/1927
this just adds an installed_profiles section.

File: src/rhsm/profile.py [hunk 1]
```diff
@@ -86,7 +86,8 @@ def __generate():
                             "version": module_pkg.getVersion(),
                             "context": module_pkg.getContext(),
                             "arch": module_pkg.getArch(),
-                            "profiles": [profile.getName() for profile in module_pkg.getProfiles()]
+                            "profiles": [profile.getName() for profile in module_pkg.getProfiles()],
+                            "installed_profiles": modules.getInstalledProfiles(module_pkg.getName())
```
Comment (jirihnidek) [inline src/rhsm/profile.py]: How should I understand to this PR? Is `profiles` useless for you and you need content provided in `installed_profiles`. I'm not against adding another item to this list.

Comment (parthaa) [reply src/rhsm/profile.py]: while ```profiles``` itself is not as useful as ```installed_profiles``` I am ok with having that information as an extra.

Comment (cnsnyder) [reply src/rhsm/profile.py]: @jirihnidek @parthaa This looks good to me. Anything left before we merge?

Comment (parthaa) [reply src/rhsm/profile.py]: nope I am happy with this :)

**Pull request merged**